### PR TITLE
ESCP-1292: Make dates strongly typed

### PR DESCRIPTION
**Pull request by shabtaisharon**

### CODE REVIEW

File: SpectraLogic.SpectraRioBrokerClient.Test/SpectraRioBrokerClientTest.cs [hunk 1]
```diff
@@ -72,8 +72,8 @@ public void ArchiveTest()
             Assert.AreEqual(JobType.ARCHIVE, job.JobType);
             Assert.AreEqual(1, job.NumberOfFiles);
             Assert.AreEqual(1234, job.TotalSizeInBytes);
-            Assert.AreEqual("2018-01-23T03:52:46.869Z[UTC]", job.CreationDate);
-            Assert.AreEqual("2018-01-23T03:52:46.869Z[UTC]", job.LastUpdated);
+            Assert.AreEqual("1/23/2018 3:52:46 AM", job.CreationDate.ToString());
```
Comment (rpmoore) [inline SpectraLogic.SpectraRioBrokerClient.Test/SpectraRioBrokerClientTest.cs]: I'm confused by this updated date format.  Why is it different?

Comment (shabtaisharon) [reply SpectraLogic.SpectraRioBrokerClient.Test/SpectraRioBrokerClientTest.cs]: before we expected the string value that returns from Rio, now we expected the ToString that DateTime return

Comment (rpmoore) [reply SpectraLogic.SpectraRioBrokerClient.Test/SpectraRioBrokerClientTest.cs]: Oh I see.  Is it still parsing the correct format from Rio though?

Comment (shabtaisharon) [reply SpectraLogic.SpectraRioBrokerClient.Test/SpectraRioBrokerClientTest.cs]: The actual parsing occurred in the new extension function

**Pull request merged**